src/pages/basic_data/app_admin/CustomerInstitutionMetadataForm.tsx

```diff
@@ -221,6 +231,30 @@ export const CustomerInstitutionMetadataForm = ({
               </Field>
             </div>
 
+            <Divider />
+
+            <Field name={CustomerInstitutionFieldNames.InactiveFrom}>
+              {({ field }: FieldProps<string | null | undefined>) => (
+                <FormControlLabel
+                  label={t('basic_data.institutions.institution_is_inactive')}
+                  control={
+                    <Checkbox
+                      onChange={(_event, checked) => {
+                        setFieldValue(
+                          CustomerInstitutionFieldNames.InactiveFrom,
+                          checked ? new Date(Date.now()).toISOString() : null
+                        );
+                      }}
```

Comment:
Ser poenget, ellers bedre at noen med faktisk React kompetanse tar QA :)

Reply:
ingen sterke meninger om date vs bool, men heller mot bool sånn rent intuitivt. Men ikke nok til at det trengs endres for min del iaf. Kan få noen rare tilfeller om man trykker på checkboxen to ganger og lagrer da, siden dataen da vil si `inactiveFrom: <1 sec ago>`, men i realiteten har den alltid vært inaktiv. Så datoen kan ikke brukes til noe annet enn en bool uansett

Reply:
Så lenge timezone havner med i responsen vil java instant parse det til sin Instant klasse, så det er ikke så farlig hvis de sitter i Japan og setter den. Men skal se om jeg klarer å få inn at vi skal ha norsk time-zone så er det lettere å lese den ut for mennesker.